R/bambu-processReads_scoreReadClasses.R

```diff
@@ -117,27 +117,28 @@ getGeneScore = function(rowData, defaultModels, fit = TRUE){
         geneModel = fitXGBoostModel(labels.train=geneFeatures$labels,
         data.train=as.matrix(features), show.cv=FALSE)
         geneScore = as.numeric(predict(geneModel, as.matrix(features)))
-        geneFDR = calculateFDR(geneScore, geneFeatures$labels)
+        #redundant function call since NDR is now calculated in extendAnno
+        geneNDR = calculateNDR(geneScore, geneFeatures$labels)
     } else {
-        warning("Gene Model not trained. Using pre-trained models, the FDR might not be adjusted correctly")
+        warning("Gene Model not trained. Using pre-trained models, the NDR might not be adjusted correctly")
         geneScore = as.numeric(predict(defaultModels$geneModel, 
                                        as.matrix(features)))
-        geneFDR = 1-geneScore
+        geneNDR = 1-geneScore
     }
     geneRCMap = match(rowData$GENEID, geneFeatures$GENEID)
     geneScore = geneScore[geneRCMap]
-    geneFDR = geneFDR[geneRCMap]
-    return(data.frame(geneScore = geneScore, geneFDR = geneFDR))   
+    geneNDR = geneNDR[geneRCMap]
+    return(data.frame(geneScore = geneScore, geneNDR = geneNDR))   
 }
 
-#' calculates the minimum FDR for each score 
-calculateFDR = function(score, labels){
+#' calculates the minimum NDR for each score 
+calculateNDR = function(score, labels){
```

Comment:
this function now should be in a different file? and the NDR calculation should be done after extendannotations, but not here anymore?

Reply:
Resolved in ce1ca659d2c8986c9a4fa0345d2e9fad9b1a6cf5